ui/src/domain/credential.ts

```diff
@@ -47,3 +47,21 @@ export interface SingleChoiceCredentialFormAttribute {
   type: "singlechoice";
   value: number;
 }
+
+export interface LinkAttributes {
+  name: string;
+  value: string;
+}
+
+export type LinkStatus = "all" | "active" | "inactive" | "exceeded";
+
+export interface Link {
+  active: boolean;
+  attributes: LinkAttributes[];
+  expiration?: Date;
+  id: string;
+  issuedClaims: number;
+  maxIssuance?: number;
+  schemaType: string;
+  status: Exclude<LinkStatus, "all">;
```

Comment:
Instead of Excluding `all` here, just remove `all` from the type `LinkStatus`.
In the view, we can use `undefined` to signify the `all` filter, which actually means "no filter" (i.e, `undefined`)